### PR TITLE
replaced all with download_regions

### PR DESCRIPTION
**Pull request by abossard**
So you can decide exactly which regions have to be downloaded..

### CODE REVIEW

**Comment by abossard** [comment]
What do you think? Personally I don't use all, but only a specific region.

File: FunKiiU.py [hunk 1]
```diff
@@ -302,12 +302,12 @@ def process_title_id(title_id, title_key, name=None, region=None, output_dir=Non
...
 
-def main(titles=None, keys=None, onlinekeys=False, onlinetickets=False, download_all=False, output_dir=None,
+def main(titles=None, keys=None, onlinekeys=False, onlinetickets=False, download_regions=[], output_dir=None,
          retry_count=3, patch_demo=True, patch_dlc=True, simulate=False):
     print('*******\nFunKiiU by cearp and the cerea1killer\n*******\n')
     titlekeys_data = []
 
-    if download_all and (titles or keys):
+    if download_regions and (titles or keys):
         print('If using \'-all\', don\'t give Title IDs or keys')
```
Comment (llakssz) [inline FunKiiU.py]: If using '-region'...

Comment (llakssz) [inline FunKiiU.py]: If using '-region'...

File: FunKiiU.py [hunk 2]
```diff
@@ -54,8 +54,8 @@
                     dest='patch_dlc', help='This will disable unlocking all DLC content')
 parser.add_argument('-nopatchdemo', action='store_false', default=True,
                     dest='patch_demo', help='This will disable patching the demo play limit')
-parser.add_argument('-all', action='store_true', default=False, dest='download_all',
-                    help='Downloads/gets tickets for EVERYTHING from the keyfile')
+parser.add_argument('-download_regions', nargs="?", default=['USA', 'EUR', 'JPN'], dest='download_regions',
```
Comment (llakssz) [inline FunKiiU.py]: I think '-region' is better, easier, I don't like the underscore in the argument that the user has to type.
But only a minor issue!

Comment (abossard) [inline FunKiiU.py]: I fully agree!

**Comment by abossard** [comment]
Changed it to -region.

**Comment by llakssz** [comment]
I tested and I saw a few problems with -region stuff, but I fixed it and I think everything is safe now.
I hope!

**Comment by abossard** [comment]
Ok, thanks! It worked perfectly on my machine, I downloaded the whole EUR
region.

**Pull request merged**